pubspec.yaml

```diff
@@ -5,7 +5,8 @@ publish_to: 'none'
 version: 0.6.0+26
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: "2.14.4" # Dart version
```

Comment:
J'ai l'impression que ça casse la CI ça 🤔 
```
The current Dart SDK version is 2.13.0.


Because pass_emploi_app requires SDK version 2.14.4, version solving failed.
```